### ISSUE TITLE
ivy-completing-read does not handle a list of defaults

### ISSUE THREAD

**Issue by darkfeline** [abo-abo/swiper#1208]
Consider:

``` emacs-lisp
(ivy-completing-read "test" '("foo" "bar") nil nil nil nil '("spam" "eggs"))
```

Both `spam` and `egg` should be added to the candidates.

**Comment by abo-abo** [comment]
Any relevant code using the defaults list?

**Comment by darkfeline** [comment]
`dired` commands do, for example `dired-do-rename` sets three defaults: Dired open in other window, current directory, and current file.

**Comment by basil-conto** [comment]
Was this fixed by #1049?

**Comment by abo-abo** [comment]
Yes, thanks.
